Samples/excel-shared-runtime-scenario/package.json

```diff
@@ -21,8 +21,8 @@
     "@babel/core": "^7.6.3",
     "@babel/polyfill": "^7.6.0",
     "@babel/preset-env": "^7.6.0",
-    "@types/office-js-preview": "^1.0.85",
     "@types/custom-functions-runtime": "^1.5.0",
+    "@types/office-js-preview": "^1.0.85",
```

Comment:
Isn't this still trying to use the preview types (instead of the GA office-js)?

Reply:
Thank you for catching that, I will fix it!

Reply:
@AlexJerabek we'll  need to leave it preview for now until the new type update rolls out to office-js.